api/views/search.py

```diff
@@ -0,0 +1,30 @@
+from api import app, db
+from api.models import Story, StoryPOI, POI
+from api.utils import create_response, row_constructor
+from flask import Blueprint, request
+
+SEARCH_URL = "/search"
+
+mod = Blueprint('search', __name__)
+
+@app.route(SEARCH_URL, methods=['GET'])
+def search_pois():
+    data = request.args
+    pois = []
+    pois1 = []
+    pois2 = []
+
+    if data.get('q') is not None:
```

Comment:
we should also send an appropriate error message for invalid requests (if `q` or `name`/`description` isn't specified)